#!/usr/bin/env python
PACKAGE = "ras_carla"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_vidion_range", double_t, 0, "maximum distance to obstacles", 100.0, 0.0, 200.0)
gen.add("min_vidion_range", double_t, 0, "minimum distance to obstacles", 3.0, 0.0, 50.0)
gen.add("ego_name", str_t, 0, "ego_vehicle name in carla world", "ego_vehicle")
gen.add("keep_time", double_t, 0, "acceleration coefficient", 2.0, 0.0, 10.0)
gen.add('conservative', bool_t, 0, 'start conservative mode', False)

exit(gen.generate(PACKAGE, "ras_carla", "ras"))
